sympy/sets/tests/test_sets.py

```diff
@@ -632,7 +632,7 @@ def test_finite_basic():
     assert (A <= B) is False
     assert AorB > A and AorB > B
     assert AorB >= A and AorB >= B
-    assert A >= A and A <= A
+    assert A >= A >= A
```

Comment:
This test is explicitly testing <=. It should not be changed.

Reply:
This test should be left as it is.